src/py/flwr/client/message_handler/message_handler_test.py

```diff
@@ -14,6 +14,7 @@
 # ==============================================================================
 """Client-side message handler tests."""
 
+import uuid
```

Comment:
We recently started to leave two empty lines between the module docstring and the first import:
```suggestion

import uuid
```